__tests__/app.test.js

```diff
@@ -5,8 +5,8 @@ const seed = require("../db/seeds/seed");
 const testData = require("../db/data/test-data");
 
 const emptyReviewsTable = async () => {
-  await db.query('TRUNCATE reviews RESTART IDENTITY CASCADE;');
-}
+  await db.query("TRUNCATE reviews RESTART IDENTITY CASCADE;");
```

Comment:
This is a fastest way to remove all the rows. TRUNCATE removes all the rows from a reviews table and as reviews table has associated relation to a comments table so I had to use additional option RESTART IDENTITY CASCADE to reset it.

I used emptyReviewsTable() to test when there is no reviews in DB and that it responds with an empty array, rather than anything else.

TDD <3